kymatio/scattering3d/core/scattering3d.py

```diff
@@ -1,7 +1,7 @@
 # Authors: Louis Thiry, Georgios Exarchakis
 # Scientific Ancestry: Louis Thiry, Georgios Exarchakis, Matthew Hirn, Michael Eickenberg
 
-def scattering3d(x, filters, rotation_covariant, L, J, max_order, backend, averaging):
+def scattering3d(x, filters, rotation_covariant, L, J, max_order, backend, averaging, vectorize=True):
```

Comment:
Leftover from other PR? I can remove it.

Reply:
Well, that's fine.

Reply:
Wow gosh, sorry, I didn't catch what you meant